### PR TITLE
Modify Japanese

### PR DESCRIPTION
**Pull request by hiroppy**
Hello :)

I modified Japanese.

cheers.

### CODE REVIEW

**Comment by dougwilson** [comment]
ping @blakeembrey

**Comment by crandmck** [comment]
Also @setogit PTAL

File: ja/advanced/best-practice-performance.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Node アプリケーションは、キャッチされていない例外が発生
 
 実行しては*ならない* ことの 1 つは、例外がイベント・ループまでたどり着いた場合に生成される `uncaughtException` イベントを listen することです。`uncaughtException` のイベント・リスナーを追加すると、例外が発生したプロセスのデフォルトの動作が変更されます。プロセスは、例外に関係なく実行し続けます。この方法でアプリケーションの異常終了を防止できそうに思えますが、キャッチされていない例外が発生した後にアプリケーションの実行を続けるのは危険な手法であり、お勧めしません。プロセスの状態の信頼性と予測可能性が低くなるためです。
 
-さらに、`uncaughtException` の使用は、正式に[粗雑なもの](https://nodejs.org/api/process.html#process_event_uncaughtexception)として認められており、これをコアから削除するための[提案](https://github.com/nodejs/node-v0.x-archive/issues/2582)が出されています。したがって、`uncaughtException` を listen するのは悪い方法です。この理由から、複数のプロセスとスーパーバイザーなどの使用をお勧めしています。異常終了と再始動は、場合によってはエラーから復旧するための最も信頼できる方法となります。
+さらに、`uncaughtException` の使用は、正式に[粗雑なもの](https://nodejs.org/api/process.html#process_event_uncaughtexception)として認められており、これをコアから削除するための[提案](https://github.com/nodejs/node-v0.x-archive/issues/2582)が出されています。したがって、`uncaughtException` を listen するのは悪い方法です。この理由から複数のプロセスとスーパーバイザーなどの使用をお勧めしています。異常終了と再始動は、場合によってはエラーから復旧するための最も信頼できる方法となります。
```
Comment (Setogit) [inline ja/advanced/best-practice-performance.md:96]: Again, the original uses the Japanese comma in the right way.  Removing it does not make a major downgrade either.

File: ja/advanced/best-practice-performance.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Node.js 4.0+ または io.js 2.1.0+ を使用している場合、アプリケ
 
 開発環境では、静的ファイルを提供するために [res.sendFile()](/{{ page.lang }}/4x/api.html#res.sendFile) を使用できます。しかし、この関数を実稼働環境では使用しないでください。この関数は、ファイル要求ごとにファイル・システムから読み取る必要があり、著しい遅延を発生させ、アプリケーション全体のパフォーマンスに影響を与えるためです。効率をはるかに向上させることができる [sendfile](http://linux.die.net/man/2/sendfile) システム呼び出しで `res.sendFile()` は実装*されない*点に注意してください。
 
-代わりに、Express アプリケーションにファイルを提供するために最適化された [serve-static](https://www.npmjs.com/package/serve-static) ミドルウェア (または同等の関数) を使用してください。
+代わりに Express アプリケーションにファイルを提供するために最適化された [serve-static](https://www.npmjs.com/package/serve-static) ミドルウェア (または同等の関数) を使用してください。
```
Comment (Setogit) [inline ja/advanced/best-practice-performance.md:58]: The original uses the Japanese comma in the right way. Removing it does not make a major downgrade either.
 Add a line note

File: ja/advanced/best-practice-performance.md [hunk 3]
```diff
@@ -131,7 +131,7 @@ app.get('/search', function (req, res) {
 
 #### Promise の使用
 
-Promise は、`then()` を使用する非同期コード・ブロックのすべての例外 (明示的と暗黙的の両方) を処理します。単に、Promise チェーンのサイトに `.catch(next)` を追加してください。次に例を示します。
+Promise は、`then()` を使用する非同期コード・ブロックのすべての例外 (明示的と暗黙的の両方) を処理します。単に、Promise チェーンの最後に `.catch(next)` を追加してください。次に例を示します。
```
Comment (Setogit) [inline ja/advanced/best-practice-performance.md:134]: Nice catch.

File: ja/advanced/best-practice-performance.md [hunk 4]
```diff
@@ -420,7 +420,7 @@ $ sudo /sbin/initctl start strong-pm
 
 重要: アプリケーション・インスタンスは別々のインスタンスとして実行されるため、同じメモリー・スペースを共有しません。つまり、オブジェクトは、アプリケーションの各インスタンスに対してローカル側にあります。そのため、アプリケーション・コードの状態を維持できません。ただし、[Redis](http://redis.io/) などのメモリー内のデータ・ストアを使用して、セッション関連のデータと状態を保管できます。この注意点は、複数のプロセスまたは複数の物理サーバーのどちらを使用したクラスタリングでも、基本的にあらゆる形式の水平スケーリングに適用されます。
 
-クラスター・アプリケーションでは、ワーカー・プロセスは、残りのプロセスに影響を与えることなく、個々に異常終了することがあります。パフォーマンス上の利点のほか、障害分離は、アプリケーション・プロセスのクラスターを実行するもう 1 つの理由です。ワーカー・プロセスが異常終了するたびに、必ず、イベントをログに記録して、cluster.fork() を使用して新規プロセスを作成してください。
+クラスター・アプリケーションでは、ワーカー・プロセスは、残りのプロセスに影響を与えることなく、個々に異常終了することがあります。パフォーマンス上の利点の他に障害分離は、アプリケーション・プロセスのクラスターを実行するもう 1 つの理由です。ワーカー・プロセスが異常終了するたびに、必ず、イベントをログに記録して、cluster.fork() を使用して新規プロセスを作成してください。
```
Comment (Setogit) [inline ja/advanced/best-practice-performance.md:423]: The original uses the Japanese comma in the right way. Changing it as suggested does not make a major downgrade either.

File: ja/advanced/best-practice-security.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ lang: ja
 
 「*実稼働*」という用語は、ソフトウェアのライフサイクルにおいて、アプリケーションや API をエンド・ユーザーまたはコンシューマーが広く使用できる段階を指します。対照的に、「*開発*」段階では、まだコードの作成とテストを積極的に行っていて、アプリケーションへの外部アクセスは不可能です。対応するシステム環境は、それぞれ*実稼働* 環境と*開発* 環境と呼ばれています。
 
-開発環境と実稼働環境は通常、別々にセットアップされ、それぞれの要件は大きく異なっています。開発環境では許可されることが、実稼働環境では許可されないことがあります。例えば、開発環境では、デバッグのためにエラーの詳細なロギングを実行できますが、同じ動作が実稼働環境ではセキュリティー上の問題となります。開発環境では、スケーラビリティー、信頼性、パフォーマンスについて心配する必要はありませんが、それらは実稼働環境では重大な問題となります。
+開発環境と実稼働環境は通常、別々にセットアップされ、それぞれの要件は大きく異なっています。開発環境では許可されることが、実稼働環境では許可されないことがあります。例えば、開発環境ではデバッグのためにエラーの詳細なロギングを実行できますが、同じ動作が実稼働環境ではセキュリティー上の問題となります。開発環境では、スケーラビリティー、信頼性、パフォーマンスについて心配する必要はありませんが、それらは実稼働環境では重大な問題となります。
```
Comment (Setogit) [inline ja/advanced/best-practice-security.md:14]: Slightly better nuance.  Removing the Japanese comma does not make a major downgrade either.

File: ja/advanced/best-practice-security.md [hunk 6]
```diff
@@ -95,7 +95,7 @@ Cookie を介してアプリケーションが悪用されないように、デ
 
 デフォルトのセッション Cookie 名を使用すると、アプリケーションが攻撃を受けやすくなります。提起されているセキュリティー問題は `X-Powered-By` と似ています。潜在的なアタッカーがこれを使用して、サーバーに対して指紋認証し、攻撃の標的にする可能性があります。
 
-この問題を回避するには、汎用 Cookie 名を使用します。例えば、[express-session](https://www.npmjs.com/package/express-session) ミドルウェアを使用します。
+この問題を回避するには、汎用な Cookie 名を使用します。例えば、[express-session](https://www.npmjs.com/package/express-session) ミドルウェアを使用します。
```
Comment (Setogit) [inline ja/advanced/best-practice-security.md:98]: The original is not an error per se. 「 汎用的な」 is better than the suggestion.  It does not hurt either though.

File: ja/advanced/pm.md [hunk 7]
```diff
@@ -112,7 +112,7 @@ $ slc ctl restart my-app
 </code>
 </pre>
 
-「ソフト再始動」も実行できます。こうすると、ワーカー・プロセスに既存の接続をクローズするための猶予期間を与えた後で、現行のアプリケーションが再始動されます。
+「ソフト再始動」も実行できます。実行するとワーカー・プロセスに既存の接続をクローズするための猶予期間を与えた後で、現行のアプリケーションが再始動されます。
```
Comment (Setogit) [inline ja/advanced/pm.md:115]: The original is not an error per se.  It does not hurt either.

File: ja/advanced/pm.md [hunk 8]
```diff
@@ -144,7 +144,7 @@ $ [sudo] npm install pm2 -g
 
 ### 基本的な使用法
 
-`pm2` コマンドを使用してアプリケーションを開始する場合、アプリケーションのパスを指定する必要があります。ただし、アプリケーションの停止、再始動、または削除を行う場合は、アプリケーションの名前または ID を指定するだけですみます。
+`pm2` コマンドを使用してアプリケーションを開始する場合、アプリケーションのパスを指定する必要があります。ただし、アプリケーションの停止、再始動、または削除を行う場合は、アプリケーションの名前または ID を指定するだけで済みます。
```
Comment (Setogit) [inline ja/advanced/pm.md:147]: The original is not an error per se. It does not hurt either.

File: ja/advanced/pm.md [hunk 9]
```diff
@@ -161,7 +161,7 @@ $ pm2 start app.js
 
 `pm2` コマンドを使用してアプリケーションを開始する場合、アプリケーションは即時にバックグラウンドに送信されます。さまざまな `pm2` コマンドを使用して、コマンド・ラインからバックグラウンド・アプリケーションを制御できます。
 
-`pm2` コマンドを使用してアプリケーションが開始された後、アプリケーションは、ID を使用して PM2 のプロセス・リストに登録されます。そのため、システム上の別のディレクトリーから、ID を使用して、同じ名前を持つアプリケーションを管理できます。
+`pm2` コマンドを使用してアプリケーションが開始された後、アプリケーションはID を使用して PM2 のプロセス・リストに登録されます。そのため、システム上の別のディレクトリーからID を使用して同じ名前を持つアプリケーションを管理できます。
```
Comment (Setogit) [inline ja/advanced/pm.md:164]: The original is not an error per se. The suggested change does not hurt either.

File: ja/advanced/security-updates.md [hunk 10]
```diff
@@ -24,8 +24,8 @@ Node.js の脆弱性は Express に直接影響を与えます。そのため、
   * 4.8.4
     * Node.js 0.10 は、特定の状況で `fd` をリークして、`express.static` および `res.sendfile` に影響を及ぼす可能性があります。悪意ある要求によって `fd` がリークされ、最終的に `EMFILE` エラーが発生したり、サーバーが応答しなくなったりする可能性があります。
   * 4.8.0
-    * 照会ストリングに極めて多数の索引が含まれる疎配列により、プロセスがメモリー不足になり、サーバーが異常終了する可能性があります。
-    * 過度にネストされた照会ストリング・オブジェクトにより、プロセスがサーバーをブロックして、サーバーが一時的に応答できなくなる可能性があります。
+    * クエリストリングに極めて多数の索引が含まれる疎な配列により、プロセスがメモリー不足になり、サーバーが異常終了する可能性があります。
```
Comment (Setogit) [inline ja/advanced/security-updates.md:27]: The original is not an error per se. The suggested change does not hurt either.

File: ja/advanced/security-updates.md [hunk 11]
```diff
@@ -38,7 +38,7 @@ Node.js の脆弱性は Express に直接影響を与えます。そのため、
   * 3.16.6
     * Node.js 0.10 は、特定の状況で `fd` をリークして、`express.static` および `res.sendfile` に影響を及ぼす可能性があります。悪意ある要求によって `fd` がリークされ、最終的に `EMFILE` エラーが発生したり、サーバーが応答しなくなったりする可能性があります。
   * 3.16.0
-    * 照会ストリングに極めて多数の索引が含まれる疎配列により、プロセスがメモリー不足になり、サーバーが異常終了する可能性があります。
-    * 過度にネストされた照会ストリング・オブジェクトにより、プロセスがサーバーをブロックして、サーバーが一時的に応答できなくなる可能性があります。
+    * クエリストリングに極めて多数の索引が含まれる疎配列により、プロセスがメモリー不足になり、サーバーが異常終了する可能性があります。
+    * 過度にネストされたクエリストリング・オブジェクトにより、プロセスがサーバーをブロックして、サーバーが一時的に応答できなくなる可能性があります。
   * 3.3.0
-    * サポートされていないメソッドのオーバーライドの試行の 404 応答は、クロスサイト・スクリプティングの攻撃を受ける可能性がありました。
+    * サポートされていないメソッドのオーバーライドの 404 応答は、クロスサイト・スクリプティングの攻撃を受ける可能性がありました。
```
Comment (Setogit) [inline ja/advanced/security-updates.md:44]: The original is not an error per se. The suggested change does not hurt either.

File: ja/index.md [hunk 12]
```diff
@@ -25,7 +25,7 @@ lang: ja
 
   <div id="boxes" class="clearfix">
       <div id="web-applications">
-          <h3>Web アプリケーション</h3> Express は、Web アプリケーションとモバイル・アプリケーション向けの一連の堅固な機能を提供する最小限の柔軟な Node.js Web アプリケーション・フレームワークです。</div>
+          <h3>Web アプリケーション</h3> Express は、Web アプリケーションとモバイル・アプリケーション向けの一連の堅固な機能を提供する最小限で柔軟な Node.js Web アプリケーション・フレームワークです。</div>
```
Comment (Setogit) [inline ja/index.md:28]: The original is not an error per se. The suggested change is okay.

File: ja/starter/faq.md [hunk 13]
```diff
@@ -52,7 +52,7 @@ app.use(function(req, res, next) {
 
 ## どのようにしてエラー・ハンドラーをセットアップするのですか?
 
-エラー処理ミドルウェアの定義方法は、他のミドルウェアと同じですが、引数の数が 3 つではなく 4 つである点が例外です。具体的には、シグニチャー `(err、req、res、next)` です。
+エラー処理ミドルウェアの定義方法は、他のミドルウェアと同じですが、引数の数が 3 つではなく 4 つである点が異なります。具体的には、シグニチャー `(err、req、res、next)` です。
```
Comment (Setogit) [inline ja/starter/faq.md:55]: Good suggestion

File: ja/starter/installing.md [hunk 14]
```diff
@@ -7,7 +7,7 @@ lang: ja
 
 # インストール
 
-既に [Node.js](https://nodejs.org/) をインストールしてあることを想定して、ここではアプリケーションを保持するディレクトリーを作成して、それを作業ディレクトリーにします。
+既に [Node.js](https://nodejs.org/) をインストールしてあることを想定して、ここではアプリケーションを作成するディレクトリーを作り、それを作業ディレクトリーにします。
```
Comment (Setogit) [inline ja/starter/installing.md:10]: The original seems slightly better.  The suggested change is not bad either.

File: ja/starter/installing.md [hunk 15]
```diff
@@ -34,7 +34,7 @@ entry point: (index.js)
 </code>
 </pre>
 
-`app.js`、またはメインファイルに指定したい名前を入力します。`index.js` にする場合は、RETURN キーを押して、示されたデフォルトのファイル名を受け入れます。
+`app.js`、またはメインファイルに指定したい名前を入力します。`index.js` にする場合は、RETURN キーを押して、表示されたデフォルトのファイル名を受け入れます。
```
Comment (Setogit) [inline ja/starter/installing.md:37]: The original seems slightly better. The suggested change is not bad either.

File: ja/starter/static-files.md [hunk 16]
```diff
@@ -7,9 +7,9 @@ lang: ja
 
 # Express での静的ファイルの提供
 
-イメージ、CSS ファイル、JavaScript ファイルなどの静的ファイルを提供するには、Express に標準装備されている `express.static` ミドルウェア関数を使用します。
+イメージ、CSS ファイル、JavaScript ファイルなどの静的ファイルを提供するには、Express に標準実装されている `express.static` ミドルウェア関数を使用します。
 
-静的資産を格納しているディレクトリーの名前を `express.static` ミドルウェア関数に渡して、ファイルの直接提供を開始します。例えば、`public` というディレクトリー内のイメージ、CSS ファイル、JavaScript ファイルを提供するには、次のコードを使用します。
+静的アセットファイルを格納しているディレクトリーの名前を `express.static` ミドルウェア関数に渡して、ファイルの直接提供を開始します。例えば、`public` というディレクトリー内のイメージ、CSS ファイル、JavaScript ファイルを提供するには、次のコードを使用します。
```
Comment (Setogit) [inline ja/starter/static-files.md:12]: Looks good.

File: ja/starter/static-files.md [hunk 17]
```diff
@@ -33,7 +33,7 @@ http://localhost:3000/hello.html
 Express は、静的ディレクトリーから相対的なファイルを検索するため、静的ディレクトリーの名前は URL の一部ではありません。
 </div>
 
-複数の静的資産ディレクトリーを使用するには、`express.static` ミドルウェア関数を複数回呼び出します。
+複数の静的アセットディレクトリーを使用するには、`express.static` ミドルウェア関数を複数回呼び出します。
```
Comment (Setogit) [inline ja/starter/static-files.md:36]: The original seems slightly better. The suggested change is not bad either.

**Comment by Setogit** [comment]
commented.  thanks.

**Comment by crandmck** [comment]
Thanks @Setogit I'll accept the PR, even though there may be a few changes that don't make much diff one way or the other.

Thanks @abouthiroppy

**Pull request merged**